src/app/monthly-calc/monthly-calc.component.ts

```diff
@@ -10,29 +10,12 @@ const fb = new FormBuilder().nonNullable;
   styleUrls: ['./monthly-calc.component.scss'],
 })
 export class MonthlyCalcComponent implements OnInit {
-  public kidsAmount = 10;
-  public applicantAmount = 2;
-
   montlhlyForm = fb.group(
     {
-      applicants: [
-        '',
-        [
-          Validators.required,
-          Validators.pattern('[0-9]*'),
-          Validators.max(this.applicantAmount),
-        ],
-      ],
-      amountOfKids: [
-        '',
-        [
-          Validators.required,
-          Validators.max(this.kidsAmount),
-          Validators.pattern('[0-9]*'),
-        ],
-      ],
+      applicants: ['', Validators.required],
+      amountOfKids: ['', Validators.required],
       income: ['', [Validators.required, Validators.pattern('[0-9]*')]],
-      obligations: ['', [Validators.required, Validators.pattern('[0-9]*')]],
+      obligations: ['', Validators.pattern('[0-9]*')],
```

Comment:
Is obligations used in template?

Reply:
<mat-form-field class="input-wrap">
        <mat-label>Obligations</mat-label>
        <input matInput formControlName="obligations" />
        <mat-error
          *ngIf="
            obligations && obligations.errors && obligations.errors['pattern']
          "
        >
          Please enter a number
        </mat-error>
        <mat-icon
          matSuffix
          matTooltip="Represent any outstanding debts or regular payments that a party must make."
          [matTooltipPosition]="'right'"
          >help_outline</mat-icon
        >
      </mat-form-field>

Reply:
` <mat-form-field class="input-wrap">
        <mat-label>Obligations</mat-label>
        <input matInput formControlName="obligations" />
        <mat-error
          *ngIf="
            obligations && obligations.errors && obligations.errors['pattern']
          "
        >
          Please enter a number
        </mat-error>
        <mat-icon
          matSuffix
          matTooltip="Represent any outstanding debts or regular payments that a party must make."
          [matTooltipPosition]="'right'"
          >help_outline</mat-icon
        >
      </mat-form-field>`

Reply:
I use line 18 to validate the input field "obligations" - if the inserted value is NOT string

Reply:
In the code from branch month-validations

Reply:
Obligations is used in lines 51-59 in template